contrib/intel/jenkins/summary.py

```diff
@@ -700,6 +700,52 @@ def check_line(self, line):
             self.check_fail(line)
             self.check_exclude(line)
 
+class DmabufSummarizer(Summarizer):
+    def __init__(self, logger, log_dir, prov, file_name, stage_name):
+        super().__init__(logger, log_dir, prov, file_name, stage_name)
+
+        self.test_type = ''
+
+    def check_type(self, line):
+        if "Running" in line:
+            self.test_type = line.split()[2]
+
+    def check_num_node(self, line):
+        if "SLURM_NNODES" in line:
+            self.num_nodes = line.split("=")[-1]
+            self.num_nodes = ' '.join([self.num_nodes, 'node'])
+
+    def check_line(self, line, log_file):
+        self.check_type(line)
+        self.check_num_node(line)
+
+    def check_name(self, line):
+        if "client_command" in line:
+            name_list = line.split()[-2:]
+            name_list.insert(0, str(self.num_nodes))
+            name_list.insert(1, str(self.test_type))
+            self.test_name = name_list
+
+    def check_pass(self, line):
+        if "TEST COMPLETED" in line:
+            self.passes += 1
+            self.passed_tests.append(self.test_name)
+
+    def check_fail(self, line):
+        if "TEST FAILED" in line:
+            self.fails += 1
+            self.failed_tests.append(self.test_name)
+
+    def read_file(self):
```

Comment:
the superclass has this organized for you already. Can you group check_line to contain "name, pass, fail, exclude" and have check_node and check_type be outside of check_line to match it?